README.md

```diff
@@ -13,3 +9,10 @@ Includes:
  - httpie
  - json
  - etc.
+
+## How it happens
+When the `master` branch of this repository is updated, CircleCI will build a new Docker image from the `Dockerfile` and it will send it to the ECR in our AWS global account.
+
+That Docker image will then be available to be used as [`executor`](https://circleci.com/docs/2.0/configuration-reference/?section=reference#executors-requires-version-21) by the CircleCI pipelines of our various projects. The ECR image tag is created by using the CircleCI job number (provided by the [built-in env variable](https://circleci.com/docs/2.0/env-vars/#built-in-environment-variables)`$CIRCLE_BUILD_NUM`).
```

Comment:
It's a version which is hard-coded in the CI config of this repo somewhere + the `CIRCLE_BUILD_NUM`

In the CI config there is this line:
```
echo 'export TAG=0.2.$CIRCLE_BUILD_NUM' >> $BASH_ENV
```

We can change the `0.2` appropriately when we make changes to this if we think it's necessary. For example, we changed it from `0.1` to `0.2` when we updated Node 10 to Node 12 on this image. That allows apps on the older version of Node to continue using `0.1.X` versions

Reply:
I'm mainly commenting to make this clear, but feel free to add something to the README if you think it's helpful